apps/test-bundles/webpackUtils.js

```diff
@@ -69,6 +70,11 @@ function createWebpackConfig(entries) {
       true,
       true,
     )[0];
+    if (packageName === '@fluentui/react-northstar') {
+      // This is used most of the configs for IE 11 compat, which northstar doesn't need
+      delete config.target;
+    }
```

Comment:
This seems a little hacky and potentially confusing if someone tries to add a `target` to northstar in the future, or if we want to configure other packages like react-components?
I'm not familiar with how this config works, but would it be possible for each package that needs ie11 compat to explicitly provide the `target` (e.g. perhaps with a public constant, `target: ie11CompatTarget`).

Feel free to ignore this comment if I'm misunderstanding how this config works -- I haven't read the code to understand how it all works.

Reply:
This config is only used for generating the bundles analyzed by size auditor (which we're about to move away from anyway). So I don't think it's a concern.

Reply:
@behowell  vNext is not supporting ie11 nor will use these  webpack configs